xml/System.Reflection.Metadata.Ecma335/MethodBodyStreamEncoder.xml

```diff
@@ -181,15 +181,15 @@
         <Parameter Name="hasDynamicStackAllocation" Type="System.Boolean" Index="6" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="codeSize">To be added.</param>
-        <param name="maxStack">To be added.</param>
-        <param name="exceptionRegionCount">To be added.</param>
-        <param name="hasSmallExceptionRegions">To be added.</param>
-        <param name="localVariablesSignature">To be added.</param>
-        <param name="attributes">To be added.</param>
-        <param name="hasDynamicStackAllocation">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="codeSize">The number of bytes to be reserved for instructions.</param>
+        <param name="maxStack">The max stack.</param>
+        <param name="exceptionRegionCount">The number of exception regions.</param>
+        <param name="hasSmallExceptionRegions"><see langword="true" /> if the exception regions should be encoded in small format; <see langword="false" /> otherwise.</param>
+        <param name="localVariablesSignature">The local variables signature handle.</param>
+        <param name="attributes">The method body attributes.</param>
+        <param name="hasDynamicStackAllocation"><see langword="true" /> if the method allocates from dynamic local memory pool (localloc instruction); <see langword="false" /> otherwise.</param>
```

Comment:
```suggestion
        <param name="hasDynamicStackAllocation"><see langword="true" /> if the method allocates from dynamic local memory pool (the <see langword="localloc" /> instruction); <see langword="false" /> otherwise.</param>
```